doc/ref_model/chapters/chapter01.md

```diff
@@ -62,7 +62,7 @@ The Reference Model specifications conform with the overall principles defined i
 
 <a name="1.4"></a>
 ## 1.4 Definitions/Terminology
-To help guide the reader, a [CNTT Terminology](../../common/glossary.md) section in [Cloud iNfrastructure Telco Taskforce] provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the [ETSI GR NFV 003 V1.5.1 (2020-01)](https://www.etsi.org/deliver/etsi_gr/NFV/001_099/003/01.05.01_60/gr_NFV003v010501p.pdf) definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
+To help guide the reader, a [CNTT Terminology](../../common/glossary.md) section in [Cloud iNfrastructure Telco Taskforce] provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003[1] definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Comment:
Minor Typo:
```suggestion
To help guide the reader, a [CNTT Terminology](../../common/glossary.md) section in Cloud iNfrastructure Telco Taskforce provides an introduction to the main terms used within this document and throughout the project in general. These definitions are, with a few exceptions, based on the ETSI GR NFV 003[1] definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```